xtts_api_server/server.py

```
@@ -1,345 +1,377 @@
+    text: str
+    speaker_wav: str 
+    language: str
+    file_name_or_path: str  
+
+@app.get("/speakers_list")
+def get_speakers():
+    speakers = XTTS.get_speakers()
+    return speakers
+
+@app.get("/speakers")
+def get_speakers():
```

Ruff (0.12.2)

Redefinition of unused `get_speakers` from line 149

(F811)